website/docs/components/application-state/partials/code/component-api.md

```diff
@@ -0,0 +1,33 @@
+## Component API
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="subtitle" @type="string">
+    The error code to be displayed. Requires both text and code (no automatic text).
```

Comment:
This explanation is a little confusing to me: "Requires both text and code"

Does it mean text for the applicable error code is required? For example: @subtitle="404"?

Reply:
Also to me it's confusing to use a more generic term like "subtitle" only for errors.